_data/remiljw/remiljw.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Michael Ajanaku 
+institution: Yaba College of Tecnology
```

Comment:
Hi @remiljw, your PR looks good and has no issues. It appears that you have misspelled "Technology" here, so I thought I'd just point it out if that's important to you.

It's not a big issue, and the PR can still be merged without making this change. It's your preference, since I saw it, I thought I'd point it out.

Reply:
Thanks, fixing now